03.bowling/bowling.rb

```diff
@@ -0,0 +1,48 @@
+# frozen_string_literal: true
+
+score = ARGV[0]
+scores = score.split(',')
+shots = []
+scores.each do |s|
+  shots << if s == 'X'
+             10
+           else
+             s.to_i
+           end
+end
+
+frame = 0
+max_frame = 10
+count = 0
+result = [0] * max_frame
+
+shots.length.times do |i|
+  if frame + 1 != max_frame
+
+    if shots[i] == 10
+      if count.zero?
+        count = 1
+        result[frame] = shots[i] + shots[i + 1] + shots[i + 2]
```

Comment:
```suggestion
        result[frame] = shots[i, 3].sum
```

みたいに直すとシンプルです。（他にも同じように直せる場所があります）

Reply:
この表記で、iから始まる3つの要素を選び取ることができるのですね！学びになりました！